gym/envs/toy_text/guessing_game.py

```diff
@@ -29,7 +29,7 @@ class GuessingGame(gym.Env):
     The agent will need to use a memory of previously submitted actions and observations
     in order to efficiently explore the available actions
 
-    The purpose is to have agents optimise their exploration parameters (e.g. how far to
```

Comment:
oofff british vs us english spelling huh :) English is not my first language, so I don't have a strong opinion either way, but in general prefer to steer clear of fixes like this. The rest looks good though, so I'll merge.